v3/tutorials/03-permissioned-network/index.mdx

```diff
@@ -149,9 +136,11 @@ First we must add the pallet to our runtime dependencies:
 **`runtime/Cargo.toml`**
 
 ```TOML
-[dependencies]
-#--snip--
-pallet-node-authorization = { default-features = false, version = '3.0.0' }
+[dependencies.pallet-node-authorization]
+default-features = false
+git = 'https://github.com/paritytech/substrate.git'
+tag = 'monthly-2021-11-1'
```

Comment:
we can tag `node-template-latest` on SUBSTRATE to use here instead.
then all template dependencies will always be `node-template-latest` and the template will be `latest`